### DISCUSSION TITLE
Any other info about ATM?

### DISCUSSION THREAD

**Discussion by MattNot** [moebiusmania/atmmi-info-ui#15] [General]
Hi!
An italian colleague here, not really a issue, I was wondering if you know if there was any more information on the status of the subway. Like crowding of any station for example.
Thank you!

**Comment by MasterCruelty** [comment]
> Hi! An italian colleague here, not really a issue, I was wondering if you know if there was any more information on the status of the subway. Like crowding of any station for example. Thank you!

Hi, yes a lot of info such as waiting time or pdf timetable, check my project "Robbot", source code `atm_feature.py` in modules/

**Comment by moebiusmania** [comment]
hey @MattNot , sorry but for some reason I've completely missed this one... so I'm a few months late 😅

this project was mostly my "case study experiment" for some stuff (most recently Nuxt.js) plus the fact that I wanted a quick way to check the metro status without the "noise" of the official website. Honestly I never looked for more features but I'm open to both proposals and PRs.

So far I'm just scraping data from ATM website and I haven't seen anything useful to add stuff in the dashboard, but again, I'm open to suggestions, I will take a look at @MasterCruelty project as he suggested to maybe get some ideas.
